lib/membrane/testing/pipeline.ex

```diff
@@ -152,33 +147,93 @@ defmodule Membrane.Testing.Pipeline do
           }
   end
 
-  @spec start_link(Options.t(), GenServer.options()) :: GenServer.on_start()
-  def start_link(pipeline_options, process_options \\ []) do
+  @type default_pipeline_keywords_list_t :: [
```

Comment:
```suggestion
  @type default_pipeline_keyword_list_t :: [
```

It's keyword list, not keywords list ;)

Reply:
indeed, done